lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1374,8 +1374,8 @@ class TextEditingChannel {
         break;
 
       default:
-        throw StateError(
-            'Unsupported method call on the flutter/textinput channel: ${call.method}');
+        EnginePlatformDispatcher.instance._replyToPlatformMessage(callback, null);
```

Comment:
We enumerate all messages in `case` clauses and make them noop. See, for example, `TextInput.setMarkedTextRect` and `TextInput.requestAutofill` above.

Reply:
Added a no-op case for the new `TextInput.setCaretRect` message.

However, I still think the default case should respond with null instead of throwing.  Text input channels do that on all other platforms, and without that change this situation will happen again the next time a new message is sent by the framework.

Reply:
By not enumerating all messages, it's hard to tell which ones are missing in the web platform. Why do other platforms fail silently? Was it a deliberate decision to do it that way?

Reply:
I assume the goal was to reduce the coupling between releases of the framework and engine.  It looks like the newly added message was only needed for one platform, and other platforms could just ignore it.

Reply:
What if we throw in debug mode and return null in profile/release modes?

Reply:
It throwing is the reason of the issue this is trying to solve.

Reply:
If other platforms ignore by default, I guess the web should too. Although I'm still not sure that's a good approach. If this was a plain Dart interface that each platform had to implement, we'd need to provide every method. Message channels make this fuzzy.

Reply:
Surely that is a valid concern and I share that opinion too. For the issue at hand however, null return should suffice in order to at least fix the breaking change.